### PR TITLE
Check in/40586/expired accordions

### PR DESCRIPTION
**Pull request by brianseek**
## Description
Adds accordions to expired error page.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#40586

## Testing done
Updated unit and e2e tests.

## Screenshots
localhost_3001_health-care_appointment-pre-check-in_error_type=expired

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/check-in/pre-check-in/pages/Error/index.jsx [hunk 1]
```diff
@@ -56,21 +57,28 @@ const Error = () => {
     </>
   );
 
-  const getErrorMessage = () => {
+  const getErrorMessages = () => {
     if (appointments && appointments.length) {
       // don't show sub message if we are 15 minutes past appointment start time
-      if (appointmentStartTimePast15(appointments)) return '';
+      if (appointmentStartTimePast15(appointments)) return ['', null];
```
Comment (brdunfield) [inline src/applications/check-in/pre-check-in/pages/Error/index.jsx:63]: Do we know for sure whether the accordions should not be present on the "past appointment start time" page as well?

Comment (brianseek) [reply src/applications/check-in/pre-check-in/pages/Error/index.jsx:63]: That is a good question, the ticket title was specific to "Expired" but I'll dig around to see if there is another ticket and look at the mocks.

Comment (brdunfield) [reply src/applications/check-in/pre-check-in/pages/Error/index.jsx:63]: From my understanding, anytime after midnight day of is expired, including 15+ min after appointment start time. Which to me means both situations should have the accordions. But I may have misunderstood what "expired" means!

Comment (brianseek) [reply src/applications/check-in/pre-check-in/pages/Error/index.jsx:63]: I think you are right. Here is the sketch: https://www.sketch.com/s/5331b114-280d-4ff5-8d36-ec49b1696b9e/p/5D647D7E-68C0-4E3B-948C-162CE11A9EB0

**Review by brdunfield — APPROVED** [review]
LGTM! 🦑

**Pull request merged**